### PR TITLE
Schema exploration query guide, new v1 functions

### LINKED ISSUE #1003: Schema exploration query guides
We need query guides that walk through exploring your schema. It should cover the following:

- [x] `buckets()` - Returns the list of buckets, which includes their retention period
- [x] `v1.measurements()` – Returns the list of measurements in a bucket
- [x] `keys()` – Returns the list of field keys in a measurement
- [x] `v1.tagKeys()`, `v1.measurementTagKeys()` - Both returns the list of keys
- [x] `v1.tagValues()`, `v1.measurementTagValues()` - Both returns the list of values

### LINKED ISSUE #1016: Document 'v1.fieldKeys' and 'v1.measurementFieldKeys' functions
PR: https://github.com/influxdata/flux/pull/2805

### PR DESCRIPTION
**Pull request by sanderson**
Closes #1003 
Closes #1016

Adds a new schema exploration query guide as well as new schema exploration functions.

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/v2.0/query-data/flux/explore-schema.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+```js
+import "influxdata/influxdb/v1"
+
+v1.tagKeys(bucket: "example-bucket")
+```
+
+### List tag keys in a measurement
+Use the [`v1.measurementTagKeys` function](/v2.0/reference/flux/stdlib/influxdb-v1/measurementtagkeys)
+to list **tag keys in a measurement**.
+_This function returns results from the last 30 days._
```
Comment (kelseiv) [inline content/v2.0/query-data/flux/explore-schema.md:106]: Maybe note you can filter, for example, if you need to find the last 5 days

File: content/v2.0/reference/flux/stdlib/influxdb-v1/fieldkeys.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+---
+title: v1.fieldKeys() function
+description: The `v1.fieldKeys()` function returns a list of fields in a bucket.
```
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/influxdb-v1/fieldkeys.md:3]: returns field keys in a bucket

File: content/v2.0/reference/flux/stdlib/influxdb-v1/fieldkeys.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+The `v1.fieldKeys()` function returns a list of fields in a bucket.
+The return value is always a single table with a single column, `_value`.
+
+```js
+import "influxdata/influxdb/v1"
+
+v1.fieldKeys(
+  bucket: "example-bucket",
+  predicate: (r) => true,
```
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/influxdb-v1/fieldkeys.md:23]: what happens if predicate isn't included?

File: content/v2.0/query-data/flux/explore-schema.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+  - /v2.0/reference/flux/stdlib/influxdb-v1/measurementtagvalues
+list_code_example: |
+  ```js
+  import "influxdata/influxdb/v1"
+
+  // List buckets
+  buckets()
+
+  // List measurements
+  v1.measurements(bucket: "example-bucket")
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/v2.0/query-data/flux/explore-schema.md:30]: This isn't for exploring v1 data. It's for replicating meta queries from v1. I've actually created a proposal on the Flux repo to separate these functions into a different package. You're not the first one to be confused by that.

File: content/v2.0/query-data/flux/explore-schema.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+---
+
+Flux provides functions that let you explore the structure and schema of your
+data stored in InfluxDB.
+
+- [List buckets](#list-buckets)
+- [List measurements](#list-measurements)
+- [List field keys](#list-field-keys)
+- [List tag keys](#list-tag-keys)
+- [List tag values](#list-tag-values)
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/v2.0/query-data/flux/explore-schema.md:50]: Listing field values is done in a normal query. These functions are meant to replicate the v1 metaquery functionality.

File: content/v2.0/query-data/flux/explore-schema.md (hunk 1 above)
Comment (sanderson) [reply content/v2.0/query-data/flux/explore-schema.md:106]: You can't customize the time range with this function. It's hardcoded to return results from the last 30 days.

File: content/v2.0/reference/flux/stdlib/influxdb-v1/fieldkeys.md (hunk 3 above)
Comment (sanderson) [reply content/v2.0/reference/flux/stdlib/influxdb-v1/fieldkeys.md:23]: The default is `(r) => true`, which means all values are returned. Nothing gets filtered out.

**Pull request merged**